kinto/deploy-test/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,5 @@
+{
+  "test_deployment.py::DeploymentTest::testChangeOfHash": true, 
+  "test_deployment.py::DeploymentTest::testCleanUp": true, 
+  "test_deployment.py::DeploymentTest::testSanityChecks": true
+}
```

Comment:
Did you mean to check in this `/.cache/` folder, or should that be in .gitignore?

**Disclaimer:** I have no idea what I'm talking about.